altsrc/flag.go

```diff
@@ -109,10 +111,14 @@ func (f *StringSliceFlag) ApplyInputSourceValue(cCtx *cli.Context, isc InputSour
 				continue
 			}
 			underlyingFlag.Value = &sliceValue
-			f.set.Set(n, sliceValue.Serialize())
+			if err := f.set.Set(n, sliceValue.Serialize()); err != nil {
```

Comment:
can you add a simple test case in altsrc for this ?

Reply:
I don't think this is reachable, as it has to be set to get to this point.